pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/UpdateTemplatesInPlace.java

```diff
@@ -196,20 +224,33 @@ private void traverseEventConfigurations(Handle handle, long studyId) {
         }
     }
 
+
     // Best-effort attempt at making a unique identifier for event configuration.
     // watch out for scenarios where diff events might end up with same unique identifier
     // ex:- trigger and order are same but diff pre/cancel expressions !
     private String hashEvent(Config eventCfg) {
-        return String.format("%s-%d",
+        String key = String.format("%s-%d",
```

Comment:
Add additional parameters so that events can be made unique and not override others